types/dexie-batch/package.json

```
@@ -1,6 +1,6 @@
 {
   "private": true,
   "dependencies": {
-    "dexie": "^2.0.0"
+    "dexie": ">1.3.6"
```

This will support Dexie < 2.0.0, too. Was that intended? And did you test the types with Dexie 3.x?

```suggestion
    "dexie": ">=2.0.0"
```

---

Supporting Dexie <2 was intentional as dexie-batch supports everything above 1.3.6 but the @typescript-bot shows that 1.x versions don't have compatible typings. That's why I think every dexie version above / or equal to v2 is okay. I tested the typings with v3 manually.